### PR TITLE
docs: add missing description to example

### PR DESCRIPTION
**Pull request by Victorsitou**
## Summary

Minor change just to be consistent and avoid this:

image

## Checklist

- [ ] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [ ] I have formatted the code properly by running `task lint`
    - [ ] I have type-checked the code by running `task pyright`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by shiftinv — APPROVED** [review]

File: changelog/888.doc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add missing description to ``dropdown.py`` example.
```
Comment (onerandomusername) [inline changelog/888.doc.rst:1]: pretty sure we don't need a changelog entry for this pr

Comment (Victorsitou) [reply changelog/888.doc.rst:1]: how will the check pass then

Comment (onerandomusername) [reply changelog/888.doc.rst:1]: skip news label

Comment (Victorsitou) [reply changelog/888.doc.rst]: 75a7811e2a43aa23657218f2b587418729ed2889

**Pull request merged**